pkg/printers/flags.go

```diff
@@ -67,7 +73,7 @@ func (f *PrintFlags) ToPrinter() (ResourcePrinter, error) {
 		return p, err
 	}
 
-	return nil, NoCompatiblePrinterError{f}
+	return nil, NoCompatiblePrinterError{Options: f, OutputFormat: f.OutputFormat}
```

Comment:
`Options: f` should include all, is it not?

Reply:
Since `OutputFormat` is a pointer, the error message was only printing the address